report/schemas.api.md

```diff
@@ -1127,6 +1127,8 @@ export type NFTFilters = {
     network?: Network;
     rentalStatus?: RentalsListingsFilterBy['status'];
     ids?: string[];
+    minPrice?: string;

```

Comment:
Should we define these new parameters as `number`? They are defined as `number` in the PR for the nft-server.